ga4gh/client.py

```diff
@@ -71,57 +200,64 @@ def searchVariants(self, request):
         is handled transparently, so that several HTTP requests may be made
         while this method executes.
         """
-        return self.runRequest(
-            request, "variants/search", protocol.GASearchVariantsResponse,
-            "variants")
+        requestContainer = self.RunRequestContainer(
+            request, "variants/search",
+            protocol.GASearchVariantsResponse, "variants")
+        return self.runRequest(requestContainer)
```

Comment:
Or maybe
```
def searchVariants(self, request):
     return self.runSearchRequest(request, "variants", protocol.GASearchVariantsResponse)
```
We should exploit the common patterns in the various protocol methods as much as possible. We know we're running a search request on 'variants' so we can build the URL appropriately and also know that the list attribute should be `variants`.